taskt.Studio/Commands/Engine Commands/SetEnginePreferenceCommand.cs

```diff
@@ -52,22 +46,23 @@ public override void RunCommand(object sender)
                     engine.AutoCalculateVariables = false;
                     break;
                 default:
-                    throw new NotImplementedException($"The preference '{preference}' is not implemented.");
+                    throw new NotImplementedException($"The preference '{v_CalculationOption}' is not implemented.");
             }
-
-
+            //TODO: Add more engine preference options
         }
+
         public override List<Control> Render(IfrmCommandEditor editor)
         {
             base.Render(editor);
 
-            RenderedControls.AddRange(CommandControls.CreateDefaultDropdownGroupFor("v_PreferenceType", this, editor));
+            RenderedControls.AddRange(CommandControls.CreateDefaultDropdownGroupFor("v_CalculationOption", this, editor));
 
             return RenderedControls;
         }
+
         public override string GetDisplayValue()
         {
-            return base.GetDisplayValue() + $" [{v_PreferenceType}]";
+            return base.GetDisplayValue() + $" [Calculation '{v_CalculationOption}']";
```

Comment:
I think it should simply be
```suggestion
            return base.GetDisplayValue() + $" [To '{v_CalculationOption}']";
```

Reply:
I think this command was intended to allow the user to set multiple engine preferences at runtime, but currently the only one available is the option to allow math to be done on variables. That's why using "To" would be too vague.

Reply:
Then it's fine.